$OpenBSD: patch-test_splice_c,v 1.1 2021/01/03 22:53:18 kn Exp $

Include <sys/select.h> to fix
  error: use of undeclared identifier 'fd_set'

Index: test/splice.c
--- test/splice.c.orig
+++ test/splice.c
@@ -23,6 +23,7 @@
 #include <errno.h>
 #include <sys/types.h>
 #include <sys/socket.h>
+#include <sys/select.h>
 #include <netdb.h>
 
 #ifdef HAVE_MINGW
